src/ray/object_manager/plasma/store.cc

```diff
@@ -892,8 +706,8 @@ bool PlasmaStore::IsObjectSpillable(const ObjectID &object_id) {
   // The lock is acquired when a request is received to the plasma store.
   // recursive mutex is used here to allow
   std::lock_guard<std::recursive_mutex> guard(mutex_);
-  auto entry = object_store_.GetObject(object_id);
-  return entry->ref_count == 1;
+  auto entry = object_lifecycle_mgr_.GetObject(object_id);
```

Comment:
What's the overhead of calling GetObject? I wonder if `object_lifecycle_mgr_ ` should just have a method to return the ref count (if there's the overhead)?